browser/password_manager/android/brave_password_manager_android_util_unittest.cc

```diff
@@ -119,9 +119,12 @@ TEST_F(BravePasswordManagerAndroidUtilTest,
 
   SetUsesSplitStoresAndUPMForLocal(pref_service(), login_db_directory());
 
-  // SetUsesSplitStoresAndUPMForLocal must not delete DB on Brave
+  // Pref should be kOff, as we want keep using profile store instead of the
+  // account store, so the paswords will be synced from Android to Desktop
   EXPECT_EQ(pref_service()->GetInteger(kPasswordsUseUPMLocalAndSeparateStores),
-            static_cast<int>(kOn));
+            static_cast<int>(kOff));
```

Comment:
we can use `base::to_underlying` instead of static_cast

Reply:
thanks, updated at
https://github.com/brave/brave-core/pull/27271/commits/df9bacb45f7655e464fef922d2d5c8ddf47e23c5